guide/C/ch_currency.xml

```diff
@@ -629,13 +623,14 @@ TODO -->
         <para>You decide to purchase stock in the Beijing Airport (Hong Kong). The ticker for this stock is
           0694.HK on Yahoo! Since you wanted to track all various income and expense amounts, here
           is the necessary account structure:
+        <itemizedlist>
+            <listitem><simpara>Assets:Investments:Brokerage Accounts:Boom:0694.HK (0694.HK)</simpara></listitem>
+            <listitem><simpara>Assets:Investments:Brokerage Accounts:Boom:Bank (HKD)</simpara></listitem>
```

Comment:
I think "Deposit" or "Cash" is better than "Bank".
But it is difficult to change its screenshot right now.

Reply:
Under Cash I understand coins and bank notes.
How about:
`<!-- ToDo Change on update of screenshot Bank -> Deposit (Type Bank) -->`

Reply:
@fellen, more English ambiguity. "Cash" is commonly used in this context to mean currency assets in a brokerage account. It used to be common in the US for these funds to be invested in a special kind of mutual fund that invested in short-term bonds and maintained a $1 NAV, but since the 2008-9 credit crisis when several of those mutual funds were either unable to maintain the NAV or failed altogether the brokerages have been pushing customers cash account into regular bank accounts. Bank isn't really a bad choice at least for US English.

Reply:
Added comment for translators.